include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_reduce.h

```diff
@@ -113,9 +113,7 @@ struct __parallel_transform_reduce_small_submitter<_Tp, __work_group_size, __ite
 
         const _Size __n_items = oneapi::dpl::__internal::__dpl_ceiling_div(__n, __iters_per_work_item);
 
-        const bool __use_usm = __use_USM_host_allocations(__exec.queue());
-        __usm_host_or_buffer_storage __res_container =
-            __usm_host_or_buffer_storage<_ExecutionPolicy, _Tp>(__exec, __use_usm, 1);
+        __usm_host_or_buffer_storage __res_container = __usm_host_or_buffer_storage<_ExecutionPolicy, _Tp>(__exec, 1);
```

Comment:
What about 
```C++
__usm_host_or_buffer_storage<_ExecutionPolicy, _Tp> __res_container(__exec, 1);
```
at this place and in all other places like this?

Reply:
Done.